frontend/taipy-gui/src/utils/formatConversion.ts

```diff
@@ -54,26 +54,20 @@ const sprintf_parse = (fmt: string) => {
     return parse_tree;
 }
 
-/**
```

Comment:
I don't know that I would have removed the comment alltogether.
It could be useful in the future...
Why not simply replate /** by /*?

Reply:
resolved